components/metal/BuildingSection.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <section class="building-section">
+    <div class="building-section__container">
+      <h2 class="building-section__title">
+        Building together
+      </h2>
+      <p class="building-section__description copy__paragraph_importance_outstanding">
+        Whether you are learning, teaching or performing scientific research, we hope Metal will get you there faster.
+      </p>
+
+      <div class="building-section__content">
+        <div class="building-section__copy">
+          <h3 class="copy__subtitle">
+            Call for community participation
+          </h3>
+          <p class="copy__paragraph">
+            This may just be the beginning, but we think Qiskit Metal has immense potential. We hope to form this software into one usable by superconducting quantum hardware designers across industry and academia, and perhaps even for other qubit architectures. Over time, we hope Metal will become the gateway for conventional microchip designers to get their feet wet designing quantum processors. And we hope that soon, quantum hardware experts will be able to write their own quantum analysis plugins and run them as their own Qiskit Metal backends.
+          </p>
+          <h3 class="copy__subtitle">
+            Open source
+          </h3>
+          <p class="copy__paragraph">
+            Qiskit Metal is a visionary work in progress in its early development stages. We're working toward a vision—a quantum design & analysis platform built from the ground up specifically for quantum hardware. It is crucial that we keep this product open source in order to grow Qiskit Metal into the software that will be most useful for quantum hardware engineers around the world and for the innovation in and growth of the field as a whole. If Qiskit Metal doesn't have what you want, then you extend Metal to build it!
+          </p>
+          <h3 class="copy__subtitle">
+            Early access
+          </h3>
+          <p class="copy__paragraph">
+            Through an early-access program, we are thrilled to ask you to join this journey to revolutionize quantum devices.
+            <br>
+            <br>
+            The early-access program will start in November and proceed through March, 2021, during which time we will work closely to develop Metal and design quantum devices with it.
+          </p>
+        </div>
+        <div class="building-section__media">
+          <img
+            class="building-section__media-img"
+            src="/images/metal/whiteboard-dark.png"
+            alt="Sketched illustration of a group standing in front of a whiteboard."
+          >
+        </div>
+      </div>
+
+      <LandingCta
```

Comment:
Instead of using `LandingCta` you can use an `AppCta`, see an example [here for the html](https://github.com/Qiskit/qiskit.org/pull/974/files#diff-f5bccdcfec2d036fb031c17293a33db1R80) and [here for the CSS](https://github.com/Qiskit/qiskit.org/pull/974/files#diff-f5bccdcfec2d036fb031c17293a33db1R321).

The reason why I prefer not to use `LandingCta` here is because it always has the arrow icon (even for external links). Look, I also created an [issue](https://github.com/Qiskit/qiskit.org/issues/973) to talk about this.